lineage/src/main/java/org/apache/gravitino/lineage/auth/AuthenticationFactory.java

```diff
@@ -32,7 +32,7 @@ public static LineageServerAuthenticationStrategy createStrategy(String authType
       case "none":
         return new NoAuthStrategy();
       default:
-        throw new BadRequestException("Unsupported authentication ");
+        throw new BadRequestException("Unsupported authentication " + String.valueOf(authType));
```

Comment:
I don't think String.valueOf is needed hare as authType is a string?

Reply:
okay. I thought whether it will be happen with nullable/empty string . let me do it like as you said.